app/main.py

```diff
@@ -1 +1,58 @@
-#  write your code here
+class Animal:
+
+    list_of_animals = []
+
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+        Animal.list_of_animals.append(self)
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = not self.is_hungry
+            return self.appetite
+        return 0
+
+
+class Cat(Animal):
+
+    def __init__(
+            self,
+            name: str,
+            is_hungry: bool = True,
+            appetite: int = 3
+    ) -> None:
+        super().__init__(name, appetite, is_hungry)
```

Comment:
**`Note: you need call the super class constructor with appetite equal to 3.`** you should set default value in super, not in constructor

Reply:
you are not right. i don't think you can put default value in super. check information about it one more time, please. and  by the way i've already cheked it, so next to check is mentor